generate devise views & update

tokuchan-pg opened the pull request:
# WHAT
サインアップ画面とサインイン画面のビューを生成し、デフォルトのものから変更する

# WHY
よりよいUI/UXを提供するため

### 作業終了後のサインアップ画面
![2017-04-15 18 53 43](https://cloud.githubusercontent.com/assets/25572309/25062743/45ea0d14-220d-11e7-98d0-99d494c07b84.png)

### 作業終了後のサインイン画面
![2017-04-15 18 53 52](https://cloud.githubusercontent.com/assets/25572309/25062744/49fd8bce-220d-11e7-8f24-d00898c7f79e.png)

tokuchan-pg committed:
generate devise views

All these 12 changes are the result of terminal command “bundle exec
rails g devise:views”

tokuchan-pg committed:
revised the view of sign-up page

tokuchan-pg committed:
revised the view of sign-in page

tokuchan-pg committed:
revised the view of partial template

tokuchan-pg committed:
applied styles to the views of sign-in and sign-up page

sato11 commented on app/assets/stylesheets/user.scss:
```diff
@@ -0,0 +1,141 @@
+    border: 1px solid #ddd;
+    border-radius: 5px;
+
+    .account-page__inner--left, .account-page__inner--right {
```
全体的にアンパサンド(&)を利用してスマートに編集してくださると尚良いです！

tokuchan-pg replied:
'&'を使ってファイル全体を編集し直しました。

kenchan0503 replied:
👍

sato11 commented on app/views/devise/confirmations/new.html.erb:
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
今回は利用しないファイルとはいえ、生成時のまま残ってしまっているのは少し気になります。

1. haml に変換する
1. 使用しないため削除する

など何かしらの意思表明を伴った操作をしてくださると開発者の意図としてわかりやすくなるのではないでしょうか。

tokuchan-pg replied:
以下のディレクトリ配下の不要なビューファイルを全て削除いたしました。ディレクトリも削除いたしました。
・app/views/devise/confirmations/
・app/views/devise/mailer/
・app/views/devise/unlocks/

kenchan0503 replied:
👍

tokuchan-pg committed:
made scss file more simple by using '&'

tokuchan-pg committed:
delete unnecessary devise view files

deleted all filed under below 3 directories
*app/views/devise/confirmations/
*app/views/devise/mailer/
*app/views/devise/unlocks/

tokuchan-pg commented:
以下２点の修正を加えました。

1. '&'を使ってapp/assets/stylesheets/user.scssのコードを簡略化

1. 以下のディレクトリ配下の不要なビューファイルを全て削除（ディレクトリも削除）
・app/views/devise/confirmations/
・app/views/devise/mailer/
・app/views/devise/unlocks/

kenchan0503 commented on app/assets/stylesheets/user.scss:
```diff
@@ -0,0 +1,141 @@
+    width: 980px;
+    margin: 60px auto;
+    padding: 60px;
+    background-color: #fff;
```
こちら複数箇所で使用しているので変数化しましょう！

tokuchan-pg replied:
変数化しました。

kenchan0503 commented on app/assets/stylesheets/user.scss:
```diff
@@ -0,0 +1,141 @@
+  .field-label {
+    .field_with_errors {
+      display: inline-block;
+      color: #d65f4c;
```
こちらも
いくつか他にもありそうなのでそちらも合わせてチェックお願いいたします

tokuchan-pg replied:
変数化しました。また、他にも変数化できる箇所がありましたので、修正致しました

kenchan0503 commented on app/views/devise/passwords/edit.html.erb:
```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
```
こちらもhamlに変換しましょう！

tokuchan-pg replied:
不要なファイルであったため、削除いたしました。

kenchan0503 commented on app/views/devise/passwords/new.html.erb:
```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
```
こちらも

tokuchan-pg replied:
不要なファイルであったため、削除いたしました。

kenchan0503 commented on app/views/devise/registrations/edit.html.erb:
```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```
こちらも

tokuchan-pg replied:
hamlにファイルを置き換えました。

tokuchan-pg committed:
Merge pull request #10 from tokuchan-pg/master

apply changes in master branch to WIP branch

tokuchan-pg committed:
made scss file more simple by using variables

tokuchan-pg committed:
revised the view of account edit page

tokuchan-pg committed:
delete unnecessary devise view files

deleted all files under app/views/devise/passwords/ and directory itself

tokuchan-pg commented:
### 以下４つの修正を新たに加えました

1. マスターブランチでの変更内容を、このブランチにマージ
1. 変数を用いてSCSSファイルを簡略化
1. アカウント編集ページ（registrations#edit）のビューをhamlに変換
1. 不要なビューファイル（app/views/devise/passwords/配下の全てのファイル）を削除

#### 上記1についての補足
以前別のブランチにて、deviseで生成したusersテーブルに自分でnameカラムを追加しましたが( https://github.com/tokuchan-pg/chat-space/pull/7 )、ストロングパラメーターの設定を失念していたため、さらに別のブランチ( https://github.com/tokuchan-pg/chat-space/pull/9 )にて対応しました。そして( https://github.com/tokuchan-pg/chat-space/pull/9 )の変更が反映されたマスターをこのブランチにマージ( https://github.com/tokuchan-pg/chat-space/pull/10 )しました。
(( https://github.com/tokuchan-pg/chat-space/pull/10 )をやらないと、サインアップを行うことができず、アカウント編集画面の変更を確認することができなかったため）

#### 上記3の画面キャプチャ（devise/registrations#edit）↓
![2017-04-17 17 51 47](https://cloud.githubusercontent.com/assets/25572309/25084291/b45a828a-2396-11e7-84ca-66857aa44102.png)
